third_party/move/move-model/src/builder/model_builder.rs

```diff
@@ -133,7 +133,7 @@ pub(crate) struct StructEntry {
 
 #[derive(Debug, Clone)]
 pub(crate) enum StructLayout {
-    Singleton(BTreeMap<Symbol, FieldData>),
+    Singleton(BTreeMap<Symbol, FieldData>, bool),
```

Comment:
Describe what the `bool` is for in a comment.

Reply:
done